Assets/Systems/StatSystem/Stats/Extentions/Vital/StatVital.cs

```diff
@@ -16,46 +16,57 @@ namespace Systems.StatSystem
     */
     public class StatVital : StatAttribute, IStatCurrentValueChanged
     {
-        private int _statCurrentValue; //the stats current value
+        private int _value; //the stats current value
 
         public event EventHandler OnCurrentValueChanged; //event for when the stat value changes  
 
         #region Constructors 
         public StatVital()
         {
-            _statCurrentValue = 0;
+            _value = 0;
         }
         #endregion
 
         #region Properties - Getters/Setters
-        public int CurrentValue
+        public int Max
+        {
+            get { return base.Value; }
+        }
+
+        new public int Value
```

Comment:
I feel like `Value` should represent the current value and not the "max value" that the `Value` of the parent classes represents.  So I decided to hide the parent's `Value` and then show it instead as a `Max` property.